server/events/command_runner.go

```diff
@@ -183,6 +185,42 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(baseRepo models.Repo, headRepo
 	}
 
 	c.updateCommitStatus(ctx, models.PlanCommand, pullStatus)
+
+	// Check if there are any planned projects and if there are any errors or if plans are being deleted
+	if len(result.ProjectResults) > 0 &&
```

Comment:
Why would we trigger policy checks? Wouldn't `ProjectResults` be empty in that case?

Reply:
nah it would still be considered a successful plan, i don't think any planfile introspection occurs in atlantis as only the plan file would say that there are no infrastructure changes.